frontend/packages/client/src/components/Community/JoinCommunityButton.js

```diff
@@ -10,18 +10,12 @@ export default function JoinCommunityButton({
 }) {
   const { createCommunityUser, deleteUserFromCommunity } = useJoinCommunity();
   const { injectedProvider, user } = useWebContext();
-  const [addr, setAddr] = useState();
-  const memberState = useUserRoleOnCommunity({
-    addr,
+
+  const isMember = useUserRoleOnCommunity({
+    addr: user?.addr,
```

Comment:
using hook output as input on another hook so we don't need create the `addr` state